linchpin/defaults/schemas/schema_v4.json

```diff
@@ -108,8 +108,35 @@
                                  "keyvalue": {
                                      "type": "array"
                                  },
-                                 "hostrequire": {
-                                     "type": "array"
+                                 "hostrequires": {
+                                     "type": "array",
+                                     "items": {
+                                         "not": {
+                                             "allOf": [
+                                                 {"required": ["force", "tag"]}
+                                             ]
+                                         },
```

Comment:
Alright, "bad" values are rejected, but the mutual exclusion stuff doesn't work. I'll fix it up.